### PR TITLE
Bug 1205608 - [v2.5][Gaia::UI Tests]test_music_share_ringtone.py:"NoSuchElementException: 

### PR DESCRIPTION
**Pull request by nojunpark**
…uchElementException: NoSuchElementException: Unable to locate element: theme-settings'"

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8d2c86706bfda8c991e2789f3da96aede6c91ac8

File: tests/python/gaia-ui-tests/gaiatest/apps/ring_tone/app.py [hunk 1]
```diff
@@ -47,7 +48,8 @@ def ring_tones(self):
 
     @property
     def screen_element(self):
-        return self.marionette.find_element(*self._screen_locator)
+        if self.is_element_present(*self._page_locator):
+            return self.marionette.find_element(*self._screen_locator)
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/apps/ring_tone/app.py]: So screen_element doesn't return anything when _page_locator doesn't exist?
Why is this change actually necessary?

Comment (npark-mozilla) [inline tests/python/gaia-ui-tests/gaiatest/apps/ring_tone/app.py]: Because screen_element refers to the ringtone list page that you access from settings, and if you access from the pick activity, this is not shown
